requirements/run.txt

```diff
@@ -1,5 +0,0 @@
-numpy==1.26
```

Comment:
don't we need all these?

Reply:
Not sure. I got this when I merged from `git pull upstream cookie`. On main, we have numpy pinned at 1.x